content/updates/2017-08-08-max-gov-migration-instructions.md

```diff
@@ -0,0 +1,45 @@
+
+## What does this mean for me?
+
+**If you already use your agency’s single sign-on (SSO) to log in to cloud.gov:** There is nothing further you need to do. Currently agencies using SSO are EPA, GSA, and FDIC. These customers will continue to log in as usual.
+
+**If you use a cloud.gov account and already have a MAX account:** Double check that you can log in successfully to MAX using a PIV/CAC card or username and password with a Secure+ SMS 2-factor device.
+
+**If you use a cloud.gov account to log in and don’t have a MAX account:** You need to create a MAX account with your PIV/CAC card or a Secure+ 2-factor device. Once we switch, you’ll be required to use that account to login to cloud.gov and all associated services.
+
+To create a MAX account:
+
+* Go to [login.max.gov](https://login.max.gov), if you see your agency's logo, click it and sign in with your agency credentials.
+* Go to [login.max.gov](https://login.max.gov), if you don't see you agency's logo, click the green Register Now button in the top right corner.
+  * Register with a PIV/CAC card if you have one and a working card reader
+  * Register with a username, password, and a Secure+ 2-factor device to your account if you don't have a card reader.
+
+### What about contractors?
+
+Contractors with a federal agency email address (.gov or .mil) should be able to self-register just like federal employees. If there are contractors or others on your team without a government email address, you’ll need to [contact OMB MAX directly](https://max.gov/maxportal/home.action) to get accounts for them.
+
+## Prepare for the switch
+
+You can create your account anytime and it only takes a few minutes. We encourage you to do it soon so you're prepared when we do switch.
+
+After the switch, you won’t be able to login with your cloud.gov account. If you don’t make a MAX account before then, your apps, orgs, and spaces will not go away but you won’t be able to login to either the dashboard or the CLI. Once you create a MAX account, you’ll regain access.
+
+## What is OMB MAX?
+
+OMB MAX is a governmentwide collaboration platform with an authentication system agencies can use to log in to other shared services. MAX.gov also enables using a PIV/CAC card to log in to web services.
+
+Though we currently use multi-factor authentication for all cloud.gov accounts, switching to MAX accounts enables agencies which require PIV/CAC cards for login to use cloud.gov more easily. MAX also makes it easier for agencies to meet their own compliance requirements for users of cloud.gov by implementing single sign-on with agency services. For most users, this will provide a better user experience by not having a separate login and second factor to keep track of for cloud.gov.
+
+We appreciate your patience and cooperation as we work to implement MAX authentication with cloud.gov.
```

Comment:
I think at the bottom here we should explain where to get help - essentially that if people have questions about logging into cloud.gov, they should email us, and if they have questions about their MAX.gov accounts (such as trouble with setting up an account or setting up 2FA) they should contact MAX.gov.